pep-0594.rst

```diff
@@ -892,15 +561,6 @@ The PyPI packages ``colormath``, ``colour``, and ``colorspacious`` provide more
 advanced features. The Pillow library is better suited to transform images
```

Comment:
The above line (one higher than I can reach with GitHub Suggestions) should also be updated to remove ``colour``, assuming it is removed from the table as suggested